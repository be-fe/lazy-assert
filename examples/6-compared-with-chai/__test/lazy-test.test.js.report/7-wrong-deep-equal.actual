{}
  b : {}
    a : "_[[[reference: @root]]]_"
    c : {}
      b : "_[[[reference: @root]]]_"
      name : "c"
    name : "b"
  name : "a"