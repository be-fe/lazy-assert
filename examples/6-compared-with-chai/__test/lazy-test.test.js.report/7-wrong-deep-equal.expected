{}
  b : {}
    a : "_[[[reference: @root]]]_"
    c : {}
      b : "_[[[reference: @root.b]]]_"
      name : "c"
    name : "b"
  name : "a"